### ISSUE TITLE
[Placeholder] PostponeUpdateIT::postpone_cancelResetsUrlOnBack

### ISSUE THREAD

**Issue by mshabarov** [vaadin/flow#19492]
### Description of the bug

PostponeUpdateIT.postpone_cancelResetsUrlOnBack is failing:
```
PostponeUpdateIT.postpone_cancelResetsUrlOnBack:76 Canceling should revert to http://localhost:8888/com.vaadin.flow.uitest.ui.PostponeUpdateView/1
```

Use case:
1. Navigate to other Flow view with postpone.
2. Do proceed
3. Navigate back with browser button
4. Cancel navigation
5. Url is not the same as it was after step 2 (this however seems to me as a wrong expectation, needs to check)

### Expected behavior

.

### Minimal reproducible example

.

### Versions

- Vaadin / Flow version:
- Java version:
- OS version:
- Browser version (if applicable):
- Application Server (if applicable):
- IDE (if applicable):

**Comment by mshabarov** [comment]
Fixed in https://github.com/vaadin/flow/pull/19505
